packages/@react-aria/menu/src/useMenuItem.ts

```diff
@@ -212,33 +207,24 @@ export function useMenuItem<T>(props: AriaMenuItemProps, state: TreeState<T>, re
 
   let {pressProps, isPressed} = usePress({
     onPressStart,
+    onPress,
     onPressUp,
-    isDisabled: isDisabled || (isTrigger && state.expandedKeys.has(key))
+    onPressChange,
+    onPressEnd,
+    // TODO: verify we actually don't need isDisabled || (isTrigger && state.expandedKeys.has(key)
```

Comment:
what do we need to check here?

Reply:
the `isDisabled` line here used to be `isDisabled: isDisabled || (isTrigger && state.expandedKeys.has(key))` but I'm fairly certain that we can get away with just `isDisabled` now. I'll remove this todo